src/components/v1/V1Project/Pay/V1PayButton.tsx

```diff
@@ -0,0 +1,137 @@
+import { t, Trans } from '@lingui/macro'
```

Comment:
**suggestion**: given this is the only file in the `Pay` directory, can probably move outside of this directory, and delete it. WDYT?

Reply:
Yep